docker/scripts/run.sh

```diff
@@ -0,0 +1,31 @@
+function build_docker_image {
+    BASENAME=$(basename $1);
+    IMAGE_NAME="newman_$BASENAME";
+    TAG="$2";
+    docker build -t "$DOCKER_ID_USER/$IMAGE_NAME:$TAG" .
```

Comment:
You'll have to add the latest tag here (-t can accept multiple space separated name:value parameters). See https://github.com/postmanlabs/newman-docker/blob/develop/scripts/docker/docker-build.sh#L13 for a working example.